projectile.el

```diff
@@ -968,23 +968,29 @@ opened through use of recentf."
   "Prepend the current project's name to STRING."
   (format "[%s] %s" (projectile-project-name) string))
 
+(defun projectile-read-buffer-to-switch (prompt)
+  "Read the name of a project buffer to switch to.
```

Comment:
This won't make `checkdoc` happy. The param name has to appear in the docstring.